test/libprojtest.pro

```diff
@@ -10,13 +10,27 @@ HEADERS += libprojtestplugin.h \
 
 # Qt Creator linking
 
+# my gcc build
 ## set the QTC_SOURCE environment variable to override the setting here
+linux-g++ {
```

Comment:
#bug
Very bad idea, to make some absolute and related to your file system paths in build system. It makes build availbale only for you, or leads others make envieronment like yours.

While it was as one just generated by qtc stub it's ok, but looks like you like to populate it.